index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
 
-  get(pos) {}
+  }
 
-  max() {}
+  get(pos) {
+    if (this.items[pos] === undefined){
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos];
+  }
 
-  min() {}
+  max() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }
 
-  sum() {}
+    return this.items[this.items.length -1];
+  }
 
-  avg() {}
+  min() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }
+
+    return this.items[0];
+  }
+
+  sum() {
+    if (this.items.length === 0){
+      return 0;
+    }
+
+    return this.items.reduce((num, acc) => num + acc);
+  }
```

Comment:
esto:

```
 if (this.items.length === 0){
      return 0;
    }
    return this.items.reduce((num, acc) => num + acc);
```

es lo mismo que esto:
`return this.items.reduce((num, acc) => num + acc, 0);`

Reply:
Ah claro! Merci :)